### PR TITLE
Remedy the madness. (email links)

### PR DESCRIPTION
**Pull request by luigiplr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,8 +71,8 @@ We expect all community participants (contributors, paid or otherwise; sponsors;
 This code of conduct and its related procedures also applies to unacceptable behavior occuring outside the scope of community activities when such behavior has the potential to adversely affect the safety and well-being of community members.
 
 ## 9\. Contact information
-- Marie Starck - @marie - marie.starck @ shaw dot ca
-- Jessica Thomas - @jess - jess.rae.thomas @ gmail dot com
+- Marie Starck - @marie - <a href="mailto:marie.starck@shaw.ca?Subject=YYJ Contect" target="_blank">marie.starck@shaw.ca</a>
+- Jessica Thomas - @jess - <a href="mailto:jess.rae.thomas@gmail.com?Subject=YYJ Contect" target="_blank">jess.rae.thomas@gmail.com</a>
```
Comment (jerath) [inline README.md]: Contect -> Contact then 👍  😃

Comment (quaelin) [inline README.md]: ...or "Connect"?

**Comment by bvanvugt** [comment]
@luigiplr What's the problem here? It's their contact information, they should display it however they want, no?

File: README.md (hunk 1 above)
Comment (luigiplr) [inline README.md]: WELL. That's awkward.

**Comment by luigiplr** [comment]
@bvanvugt Mainly to enhance the ease of utility by reducing the amount of manual clicking/typing required by the end user.

**Comment by jerath** [comment]
I stripped the email formatting because I had some concerns about email scraping. I agree that adding a `mailto` link is much easier than typing it out by hand. Thanks!

**Pull request merged**